src/space.cpp

```diff
@@ -2433,8 +2433,17 @@ void space_text(void)
             {
                column += next->orig_col - pc->orig_col_end;
             }
+            else
+            {
+               // preserve the position if virtual brace
+               // Issue #1854
+               if (chunk_is_token(pc, CT_VBRACE_OPEN))
```

Comment:
Ok, but why don't you use else if here ?

Reply:
Looks like @gmaurel forgot to reply to your question